test/gululog_topic_SUITE.erl

```diff
@@ -81,6 +82,53 @@ t_basic_flow(Config) when is_list(Config) ->
 
   ok.
 
+t_truncate_inclusive({init, Config}) ->
+  Config;
+t_truncate_inclusive({'end', _Config}) ->
+  ok;
+t_truncate_inclusive(Config) when is_list(Config) ->
+  Dir = ?config(dir),
+  BackupDir = filename:join(Dir, "backup"),
+  T1 = gululog_topic:init(Dir, []),
```

Comment:
too long sequential numbers, impossible to update
please change it to a lists:foldl etc.